public/js/editor.js

```diff
@@ -1,25 +1,25 @@
 apos.define('apostrophe-rich-text-widgets-editor', {
 
   afterConstruct: function(self) {
-    console.log(self.__meta.name);
-    console.log('>>', apos.modules[self.__meta.name.replace(/-editor$/, '')].options);
+    apos.utils.log(self.__meta.name);
```

Comment:
Functionally, it might let us control how logs are handled uniquely by Apostrophe. But practically, it helps separate "permanent" logs from temporary ones used in development. Though these do look like development logs.

Should I remove them or revert them, let the test fail, and let you clean them up before it gets to 1.x?

Reply:
Ok, thanks for the info